src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
+
+I poked him hard in the tit with a pool cue to prop his body up, and grabbed the letter from his pocket. I noticed his keys jangling on his side too.
+
+(link: "I took Ron’s keys, even the one to his shitty golf cart, and headed for the exit")[(set: $cartKeyTaken to true)(go-to: "The Gutter - Ending")].
+(link: "I took the key to the Church door, but left the key to Ron’s shitty golf cart. I made a beeline for the exit")[(set: $cartKeyTaken to false)(go-to: "The Gutter - Ending")].
+
+::The Gutter - Pool table - First approach
+The teens didn’t move an inch. In fact, they seemed to be rounding first base.
+
+[I coughed.]<poolTableCough|
+[[[I turned back and let the kids have their fun|The Gutter - Inside hub]].]<poolTableExit1|
+\(click: ?poolTableCough)[(replace: ?poolTableExit1)[$dissolve[They were still ignoring me. Clothes were coming off and the only benefit was that I got to see a tattoo of a hatchet man dressed as Jesus pissing on Satan.
+
+[I kicked the leg of the goddamn table.]<poolTableKick|
+[[[I turned back. They were getting pretty close to just fucking at this point|The Gutter - Inside hub]].]<poolTableExit2|]]]
+\(click: ?poolTableKick)[(replace: ?poolTableExit2)[$dissolve["What's your fucking problem, sis?" one of the teens said, in-between tongue thrusts. "Go ruin somebody else's shit.”]]]
+
+(click: "ruin")[$dissolve[(set: $poolTableAttemptMade to true)[I frowned in a way that made me feel very old. [[Guess these horny ass kids weren’t moving for me|The Gutter - Inside hub]].]]]
+
+::The Gutter - Pool table - Second approach
+Those teens were still twisting tongues and dry humping like the world was ending. I had to respect their stamina at the very least. This wasn't something I could [[interrupt by myself|The Gutter - Inside hub]].
+
+::The Gutter - Pool table - Clear
+Though a residual funk hung over it, the table was finally clear and [[ready for a game|The Gutter - Inside hub]].
+
+::The Gutter - Jukebox - Intro
+The Jukebox was whichever punkass MC that the Honchos had decided needed to prove their salt that week. They were supposed to take requests, learn to serve the family, but most people ignored the Jukeboxes.
+
+The kid on shift today looked like he was on break. He was just fiddling with an old laptop that crunched instead of clicked. I headed his way, but my shoulders did the aw-shit slump when he turned around and I realized it was [[Donnie|The Gutter - Jukebox - Donnie]].
+
+::The Gutter - Jukebox - Donnie
+Donnie was always doing the weirdest shit. The kid barely talked, but when he did it was to quote a song. He’d get stuck on something for months at a time and couldn’t get let it go.
+
+Anna thought he was alright, but he gave me a weird vibe. Did he hate me? Did he even notice me? Impossible to tell. Still, he could have info on Anna and that was worth a lot right now.
```

Comment:
I don't need to die on this hill, but "all right" is still the optimal spelling. If "alright" reads better to y'all, we can stick with it.

Reply:
Nah, I'll let it drop here. Thanks for humoring me. :)